libs/wingc/src/jsify.rs

```diff
@@ -394,6 +394,16 @@ impl JSifier {
 						.join(", ")
 				)
 			}
+			ExprKind::SetLiteral { items, .. } => {
+				format!(
+					"Object.freeze({{{}}})",
+					items
+						.iter()
+						.map(|expr| self.jsify_expression(expr, phase))
+						.collect::<Vec<String>>()
+						.join(", ")
+				)
+			}
```

Comment:
In the spec, Set us using the [global Set class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set), which also does not a constructor to initialize it with values so you'd have to call add for each one with something like this (untested)

```suggestion
			ExprKind::SetLiteral { items, .. } => {
				format!(
					"Object.freeze(new Set(){})",
					items
						.iter()
						.map(|expr| format!(".add({})", self.jsify_expression(expr, phase)))
						.collect::<Vec<String>>()
						.join("")
				)
			}
```

Reply:
Wait, just kidding the constructor can take an array.

```suggestion
			ExprKind::SetLiteral { items, .. } => {
				format!(
					"Object.freeze(new Set([{}]))",
					items
						.iter()
						.map(|expr| self.jsify_expression(expr, phase))
						.collect::<Vec<String>>()
						.join(", ")
				)
			}
```